### DISCUSSION TITLE
Licence vérouillée (vérifier vos emails)

### DISCUSSION THREAD

**Discussion by johnnny-debug** [aclap-dev/video-downloadhelper#1345] [License & Purchase requests]
### What's your issue?

Licence vérouillée (vérifier vos emails)

### The 10 first digits (only 10) of your license number or email address

c039a80ea5

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry, you should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Comment by johnnny-debug** [comment]
jcfrog merci
